prog/weaveutil/datapath.go

```diff
@@ -21,7 +33,24 @@ func createDatapath(args []string) error {
 		// status to distinguish it for the weave script.
 		os.Exit(17)
 	}
-	return err
+	if err != nil {
+		return err
+	}
+
+	if mtu > 1500 {
+		// Check whether a host is exposed to https://github.com/weaveworks/weave/issues/1853
+		// If yes, fallback to 1500 MTU.
+		if !checkMTU(dpname, mtu) {
+			fmt.Fprintf(os.Stderr, "Unable to set MTU to %d potentially due to 4.3 or 4.4 kernel version; Setting MTU to 1500\n", mtu)
```

Comment:
"WARNING: Unable to set MTU to %d, possibly due to 4.3/4.4 kernel version. Setting MTU to 1500 instead.\n"

(this will still look odd if it pops up when the user is actually running a different kernel)